### DISCUSSION TITLE
I can't see the 'Proof Type' options when submitting the proof of the academic status

### DISCUSSION THREAD

**Discussion by Davidtennent** [community/community#81966] [GitHub Education] [answered]
### Hi everyone,

I was trying to apply for student verification today and encountered an issue when submitting my enrollment certificate. I couldn't find the proof type options, but since I used an enrollment certificate, which belong to 'another form of proof', so after submission, I received a message instructing me to select "other" in the proof type. However, I have been unable to submit it successfully. Does anyone have a solution for this issue?

**Comment by faridnec** [comment, marked as answer]
I had similar issues then I tried to use the direct camera before submitting. It worked they accepted it. Have you tried with the camera?

**Reply by Davidtennent** [reply]
Thank you very much! After I printed the certificate and use the direct camera, the submitting finally succeeded!
